tests/valid-credential.json

```diff
@@ -0,0 +1,9 @@
+{
+  "@context": ["https://www.w3.org/2018/credentials/v1"],
+  "credentialSubject": {
+    "id": "did:example:123"
+  },
+  "issuanceDate": "2006-01-02T15:04:05Z",
+  "issuer": "did:web:example.com",
```

Comment:
nit: This won't resolve (obviously), do we want to maybe use an issuer that would resolve as a valid example?

Reply:
hmm. The goal of this request is to fail _in the right way_. If we think that an implementation might fail because the issuer cannot be resolved, then I would say yes, we should change this to something that will resolve. Suggestions welcome.

Reply:
Agree 100% - we should use did:key for the issuer.

Reply:
Issue #404 has been created to track this change.